app.js

```diff
@@ -13,6 +13,14 @@ new CronJob(cronFrequency, function() {
   request.post('http://localhost/export-tasks');
 }, null, true);
 
+new CronJob('* * * * *', async () => {
+  const retriableTasks = await getTasksThatCanBeRetried();
+  console.log(retriableTasks)
```

Comment:
Is this logged for development? If so, remove. Otherwise, add some more context to the logged line so it's clear by reading the logs what's printed in the console.